build.rs

```diff
@@ -52,42 +60,46 @@ fn build(sdk_path: Option<&str>, target: &str) {
         // Since iOS 10.0 and macOS 10.12, all the functionality in AudioUnit
         // moved to AudioToolbox, and the AudioUnit headers have been simple
         // wrappers ever since.
-        if target.contains("apple-ios") {
-            // On iOS, the AudioUnit framework does not have (and never had) an
-            // actual dylib to link to, it is just a few header files.
-            // The AudioToolbox framework contains the symbols instead.
-            println!("cargo:rustc-link-lib=framework=AudioToolbox");
-        } else {
+        if target.contains("apple-darwin") {
             // On macOS, the symbols are present in the AudioToolbox framework,
             // but only on macOS 10.12 and above.
             //
             // However, unlike on iOS, the AudioUnit framework on macOS
             // contains a dylib with the desired symbols, that we can link to
             // (in later versions just re-exports from AudioToolbox).
             println!("cargo:rustc-link-lib=framework=AudioUnit");
+        } else {
+            // On iOS, the AudioUnit framework does not have (and never had) an
+            // actual dylib to link to, it is just a few header files.
+            // The AudioToolbox framework contains the symbols instead.
+            println!("cargo:rustc-link-lib=framework=AudioToolbox");
+        }
+        if !target.contains("apple-watchos") {
+            headers.push("AudioUnit/AudioUnit.h");
         }
```

Comment:
If the headers aren't available on watchOS, maybe it would be prudent to also not (try to) link?

Reply:
+1 on watchOS